### PR TITLE
Make the introduction package available for TYPO3 11

### PR DESCRIPTION
**Pull request by linawolf**
We are planing to use the introduction package in TYPO3 11 to provide automatic screenshots for some manuals.

The introduction package seems to work fine. But we do have to fork both the introduction package and the bootstrap package to adjust their requirements so they can be included with composer.

### CODE REVIEW

**Comment by gilbertsoft** [comment]
Thanks for this. First step must be releasing a new Bootstrap Package. Afterwards I will take care of the Introduction.

**Comment by linawolf** [comment]
I made a PR there too. The Bootstrap Package already supports 11 but strangely requires it to be a dev installation...

https://github.com/benjaminkott/bootstrap_package/pull/971#pullrequestreview-590033717

File: ext_emconf.php [hunk 1]
```diff
@@ -11,9 +11,9 @@
 	'author_email' => '',
 	'constraints' => array(
 		'depends' => array(
-			'typo3' => '10.0.0-10.4.99',
+			'typo3' => '10.0.0-11.0.99',
```
Comment (bmack) [inline ext_emconf.php:14]: can you set this to 11.1.9 or ideally 11.5.99 directly?

same below

File: composer.json [hunk 2]
```diff
@@ -5,12 +5,12 @@
 	"homepage": "https://typo3.org",
 	"license": ["GPL-2.0-or-later"],
 	"require": {
-		"typo3/cms-core": "^10.0",
-		"typo3/cms-impexp": "^10.0",
-		"typo3/cms-form": "^10.0",
-		"typo3/cms-indexed-search": "^10.0",
-		"typo3/cms-felogin": "^10.0",
-		"bk2k/bootstrap-package": "~11.0.0"
+		"typo3/cms-core": "^11.0 || ^11.1",
```
Comment (bmack) [inline composer.json:8]: we only need ^11.0 here...

File: ext_emconf.php (hunk 1 above)
Comment (linawolf) [reply ext_emconf.php]: done

File: ext_emconf.php [hunk 3]
```diff
@@ -11,9 +11,9 @@
 	'author_email' => '',
 	'constraints' => array(
 		'depends' => array(
-			'typo3' => '10.0.0-10.4.99',
+			'typo3' => '10.0.0-11.5.99',
```
Comment (gilbertsoft) [inline ext_emconf.php:14]:
```suggestion
			'typo3' => '11.2.0-11.5.99',
```

File: ext_emconf.php [hunk 4]
```diff
@@ -11,9 +11,9 @@
...
 			'bootstrap_package' => '11.0.0-11.5.99',
-			'impexp' => '10.0.0-10.4.99'
+			'impexp' => '10.0.0-11.5.99'
```
Comment (gilbertsoft) [inline ext_emconf.php:16]:
```suggestion
			'impexp' => '11.2.0-11.5.99'
```

File: composer.json [hunk 5]
```diff
@@ -5,12 +5,12 @@
...
-		"typo3/cms-impexp": "^10.0",
-		"typo3/cms-form": "^10.0",
-		"typo3/cms-indexed-search": "^10.0",
-		"typo3/cms-felogin": "^10.0",
-		"bk2k/bootstrap-package": "~11.0.0"
+		"typo3/cms-core": "^11.0",
+		"typo3/cms-impexp": "^11.0",
+		"typo3/cms-form": "^11.0",
+		"typo3/cms-indexed-search": "^11.0",
+		"typo3/cms-felogin": "^11.0",
```
Comment (gilbertsoft) [inline composer.json:12]:
```suggestion
		"typo3/cms-core": "^11.2",
		"typo3/cms-felogin": "^11.2",
		"typo3/cms-form": "^11.2",
		"typo3/cms-impexp": "^11.2",
		"typo3/cms-indexed-search": "^11.2",
```

File: ext_emconf.php [hunk 6]
```diff
@@ -11,9 +11,9 @@
 	'author_email' => '',
 	'constraints' => array(
 		'depends' => array(
-			'typo3' => '10.0.0-10.4.99',
+			'typo3' => '10.0.0-11.5.99',
 			'bootstrap_package' => '11.0.0-11.5.99',
```
Comment (gilbertsoft) [inline ext_emconf.php:15]:
```suggestion
			'bootstrap_package' => '12.0.0-12.9.99',
```

**Review by gilbertsoft — CHANGES_REQUESTED** [review]

**Comment by tmotyl** [comment]
FYI, Benji has just released a v11 compatible bootstrap_package version.
https://github.com/benjaminkott/bootstrap_package/releases/tag/12.0.0

File: composer.json [hunk 7]
```diff
@@ -5,12 +5,12 @@
...
+		"bk2k/bootstrap-package": "dev-master"
```
Comment (tmotyl) [inline composer.json:13]: this can now point to 12.0.0

**Comment by tmotyl** [comment]
@linawolf can you elaborate why was it closed?

**Comment by linawolf** [comment]
In the current version we loose data on the import, so Alex and I are debugging what happens. I will make another PR once we have solved the current problems

**Comment by linawolf** [comment]
This PR is continued in https://github.com/FriendsOfTYPO3/introduction/pull/41

**Pull request closed without merging**